sdk/formrecognizer/ai-form-recognizer/test/utils/recordedClients.ts

```diff
@@ -80,6 +81,31 @@ export const environmentSetup: RecorderEnvironmentSetup = {
   queryParametersToSkip: []
 };
 
+export function createRecorder(context: Context): Recorder {
+  return record(context, environmentSetup);
+}
+
+/*
+class TestCredential implements TokenCredential {
+  public async getToken(..._: any[]): Promise<AccessToken> {
+    return {
+      token: "",
+      expiresOnTimestamp: Date.now() + 100000
+    };
+  }
+}
+*/
+
+export function makeCredential(useAad: boolean): TokenCredential | AzureKeyCredential {
+  //  if (isPlaybackMode()) {
```

Comment:
nit: remove instead of commenting out?

Reply:
Good catch thank you, I forgot to clean up some debugging changes to the utils.